### ISSUE TITLE
部署appid服务端提示部署完成 可是GAE那块 配置不了 

### ISSUE THREAD

**Issue by Freedom000** [XndroidDev/Xndroid#17]
如题 另外 一直让导入证书

**Comment by XndroidDev** [comment]
> 部署appid服务端提示部署完成
目前不支持在手机上部署APPID,请在电脑上完成.手机上填上已部署的APPID.
> 另外 一直让导入证书
你是否导入过证书,如果是Android 7.0 及以上,可能出现不信任用户导入的证书的情况,建议在可忽略证书警告的浏览器(如<内置>LightningBrowser, X浏览器), 或可导入证书的浏览器(如firefox)中使用.

**Comment by Freedom000** [comment]
是7.1的系统 那如果我电脑上完成部署的话 手机上直接填已部署的，这样可以吗？你的意思是同一个局域网内必须在电脑上部署成功，然后客户端才可以用吗？

**Comment by Freedom000** [comment]
就说电脑上必须也开着xx 然后手机也开着是吗

**Comment by XndroidDev** [comment]
部署完成后直接填上就不用管了

**Comment by XndroidDev** [comment]
部署一次即可,完成后无需电脑

**Comment by XndroidDev** [comment]
也不需要在同一个局域网

**Comment by Freedom000** [comment]
就是说只要用过xxnet 自己的id部署过，直接填到这个安卓客户端上就行是吗 那如果是这样的话，我电脑是试用过很多次的  但是我填到这个客户端上，它还是显示用的公用的appid，访问受限 第一次用的时候，google，google play都可以用，但是，youtube不能用

**Comment by XndroidDev** [comment]
是的,可能设置时出了问题,你可在XX-Net中设置APPID试试,将日志发送到xndroiddev@gmail.com,注明机型,系统版本,异常情况

**Comment by Freedom000** [comment]
好的 我再试试

**Comment by Freedom000** [comment]
邮件已发 有空可以看下 我邮箱：duanzhaoyang1017@gmail.com

**Comment by Freedom000** [comment]
重新导出的日志，邮件已回复发送到你邮箱

**Comment by imldf** [comment]
這個軟件估計有的手機品牌會有問題，HTC完全OK
